src/main.rs

```diff
@@ -314,7 +315,7 @@ mod tests {
   use std::io::Read;
 
   use crate::*;
-
+  
```

Comment:
Seems like the formatter wasn't run. Could you please follow the [instructions here](https://github.com/Eakam1007/rost_gen/blob/main/CONTRIBUTING.md) and run clippy and (nightly)fmt?

Reply:
I just pushed an update. Sorry that I missed the necessary steps.